src/main/kotlin/no/nav/helse/navfhirserver/security/SecurityConfig.kt

```diff
@@ -0,0 +1,67 @@
+package no.nav.helse.navfhirserver.security
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.http.HttpMethod
+import org.springframework.security.config.annotation.web.builders.HttpSecurity
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
+import org.springframework.security.config.annotation.web.invoke
+import org.springframework.security.oauth2.jwt.JwtDecoder
+import org.springframework.security.oauth2.jwt.NimbusJwtDecoder
+import org.springframework.security.web.SecurityFilterChain
+
+@Configuration
+@EnableWebSecurity(debug = true)
+class SecurityConfig() {
+
+    @Value("spring.security.oauth2.authorizationserver.issuer")
+    private lateinit var jwksUri: String
+
+    @Bean
+    fun securityFilterChain(http: HttpSecurity): SecurityFilterChain {
+        http {
+            authorizeHttpRequests {
```

Comment:
Her trenger vi å håndtere at en innkommende SMART launch kan forlange alle mulige varianter av scopes
`(patient|encounter|user)/[ResourceType].(c|r|u|d|s|*)` der ".cruds" suffix er 5 individuelle rettigheter, mens "*" er _full access_.

For eksempel, si en SMART on FHIR applikasjon ønsker scopes: "openid profile patient/*.rs user/*.cud"
Access token skal da gi følgende tilganger:
- openid (id token)
- profile (brukeinfo i id token)
- LESE pasient
- SØKE pasient
- OPPRETTE bruker
- OPPDATERE bruker
- SLETTE bruker

Med andre ord må scopes gjøres om til tilganger auth serveren forstår, generere access token med tilgangene.

Hvordan gjøres dette i spring security?